client/state/utils.js

```diff
@@ -260,11 +261,105 @@ export function createReducer( initialState = null, customHandlers = {}, schema
  * @returns {function} wrapped reducer handling validation on DESERIALIZE
```

Comment:
We don't match against the schema on SERIALIZE since it's expensive. We do still need to fire off `DESERIALIZE` to allow reducers to handle any needed transforms to avoid errors when we attempt to save to browser storage.